src/js/tech/html5.js

```diff
@@ -889,6 +893,9 @@ Html5.supportsNativeTextTracks = function() {
   if (supportsTextTracks && browser.IS_FIREFOX) {
     supportsTextTracks = false;
   }
+  if (supportsTextTracks && !('onremovetrack' in Html5.TEST_VID.textTracks)) {
```

Comment:
Looks like supportsNativeTextTracks could use some simplification now, but not worth blocking this.